src/wui/interactive_base.cc

```diff
@@ -752,30 +752,6 @@ void InteractiveBase::draw_overlay(RenderTarget&) {
 
 	Game* game = dynamic_cast<Game*>(&egbase());
```

Comment:
If you replace the checks for `game == nullptr` with `!egbase().is_game()` below, you can now remove the dynamic cast here